tests/manage/test_pvc_disruptive.py

```diff
@@ -0,0 +1,119 @@
+import pytest
+import logging
+from ocs_ci.ocs import constants
+from tests import helpers, disruption_helpers
+from ocs_ci.framework.testlib import ManageTest, tier4
+from tests.fixtures import (
+    create_rbd_storageclass, create_ceph_block_pool, create_cephfs_storageclass,
+    create_rbd_secret, create_cephfs_secret
+)
+
+logger = logging.getLogger(__name__)
+
+DISRUPTION_OPS = disruption_helpers.Disruptions()
+
+
+class BaseDisruption(ManageTest):
+    """
+    Base class for PVC related disruption tests
+    """
+    pod_obj = None
+    pvc_obj = None
+    storage_type = None
+
+    def disruptive_base(self, operation_to_disrupt, resource_to_delete):
+        """
+        Base function for PVC disruptive tests
+        """
+        DISRUPTION_OPS.set_resource(resource=resource_to_delete)
+        self.pvc_obj = helpers.create_pvc(
+            sc_name=self.sc_obj.name, wait=False
+        )
+        if operation_to_disrupt == 'create_pvc':
+            DISRUPTION_OPS.delete_resource()
```

Comment:
Are we able to achieve disruption during resource creation?
Anyway, we are also planning to work on a function to create multiple pvcs with random accessmodes, maybe we can use that here afterwards.

Reply:
yes, exactly, I've discussed this with Neha, please consider this PR as the base functionality for the disruptive and enhance it afterwards to have multiple PVCs creation for instance